src/kontrol/prove.py

```diff
@@ -613,6 +594,17 @@ def _init_cterm(
         init_subst['CALLGAS_CELL'] = intToken(0)
         init_subst['REFUND_CELL'] = intToken(0)
 
+    if symbolic_exploration:
+        init_subst.update(
+            {
+                'ID_CELL': Foundry.address_TEST_SYMBOLIC(),
+            }
+        )
+        del init_subst['CALLSTACK_CELL']
+        del init_subst['CALLDEPTH_CELL']
+        del init_subst['LOG_CELL']
+        del init_subst['INTERIMSTATES_CELL']
```

Comment:
These could also be removed from `init_subst` for symbolic exploration:
```
        'TOUCHEDACCOUNTS_CELL': set_empty(),
        'ACCESSEDACCOUNTS_CELL': set_empty(),
        'ACCESSEDSTORAGE_CELL': map_empty(),
```
Alternatively, they could be added in post-production of obtained specifications.